foreign_cc/cmake.bzl

```diff
@@ -19,24 +19,21 @@ load(
     "cc_external_rule_impl",
     "create_attrs",
 )
-load(
-    "//toolchains/native_tools:tool_access.bzl",
-    "get_cmake_data",
-    "get_make_data",
-    "get_ninja_data",
-)
+load("//toolchains/native_tools:tool_access.bzl", "get_cmake_data", "get_ninja_data")
 
 def _cmake_impl(ctx):
     cmake_data = get_cmake_data(ctx)
-    make_data = get_make_data(ctx)
+    ninja_path = None
 
-    tools_deps = ctx.attr.tools_deps + cmake_data.deps + make_data.deps
```

Comment:
Where is dependency on the make toolchain now if `Unix Makefiles` generator is picked?

Reply:
Good catch, restored the make_data here and added a TODO to expand on controlling the generator target being used. I think this should be it's own attribute so the rules can appropriately identify what toolchains to include in the build.

Reply:
I've actually gone ahead and added support for detecting the current generator or selecting a default. I think this is the correct thing to do.